src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/persistence/PersistorBase.java

```diff
@@ -123,8 +125,15 @@ public synchronized String read() {
     return readTables();
   }
 
-  public synchronized RcaResponse readRca(String rca) {
-    return readRcaTable(rca);
+  // TODO : readRca might return a list of RcaResponse if using range query
```

Comment:
should we just fix it now ?

Reply:
If we intend to keep this as a todo for now, we should expand the text saying what the TODO intends to solve but why this is not required immediately

Reply:
I have expanded the TODO text to explain why we didn't add range query here.  

It is still unclear to me what needs to be done here and how are we going to define the RESTful API to do range query. Since BeagleRock does not need this either at this moment we can revisit this function to decide what are the input arguments of the new API how are we going to do range query.